tests/end-to-end/tests/linechart.cy.ts

```diff
@@ -0,0 +1,49 @@
+import { selectors, testIds } from '../support/const';
+const paths = {
+lineChart: 'ComponentOverlay_rootcontainer/maincontainer/pagesection/linechart'
```

Comment:
does ComponentOverlay need to have capitals?

Reply:
It is by default in the path of a particular component.